code/game/gamemodes/gameticker.dm

```diff
@@ -48,9 +48,9 @@ var/global/datum/controller/gameticker/ticker
 
 	send2mainirc("Server lobby is loaded and open at byond://[config.serverurl ? config.serverurl : (config.server ? config.server : "[world.address]:[world.port]")]")
 
-	pregame_timeleft = config.pregame_time
 
 	do
+		pregame_timeleft = config.pregame_time
 		to_chat(world, "<B><FONT color='blue'>Welcome to the pregame lobby!</FONT></B>")
 		to_chat(world, "Please set up your character and select ready. The round will start in [pregame_timeleft] seconds.")
 		while(current_state == GAME_STATE_PREGAME)

```

Comment:
Oh, I didn't see this inside loop and thought I was being smart and moved it out of a loop that would just keep resetting it to 180. XD